### PR TITLE
section 7: generative algorithms

### PR DESCRIPTION
**Pull request by drcandacemakedamoore**
DRAFT

### CODE REVIEW

**Comment by drcandacemakedamoore** [comment]
Pending further discussion with Cunliang

**Review by gcroci2 — APPROVED** [review]
I added two very minor comments, and refined the text in commit https://github.com/esciencecenter-digital-skills/medical-image-processing/pull/21/commits/27fb0eafc7b6b739af13025592990300dc590934, but the lesson (and the examples) look great! Feel free to merged after addressing the comments :)

File: episodes/introduction_to_generative_ai.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
+---
+title: "Generative AI"
+teaching: 30
+exercises: 1
```
[the remark this answers is not in the document]
Comment (drcandacemakedamoore) [reply episodes/introduction_to_generative_ai.md:4]: Oh, oops, I get it now. I thought it was exercise number

File: episodes/introduction_to_generative_ai.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+The potential uses of such technologies are quite broad in the field of medical imaging. Potentially such tools can even be used in the interpretation of medical images as well as a plethora of other tasks. Obviously participants in this course may be particularly interested in the facts that large language models (LLMs) can be used to generate code, and some models may be able to generate vast quantities of synthetic image or text data. Research is ongoing. There is also an ongoing debate about the safety of such models. What is certain is that depending upon the software or model you use, data given into the system may become the property of the software owners. Think twice before entering any patient data into such a system. Try to understand what might happen to the data. Does it stay on your server or go into the cloud somewhere (which may be then sent to data storage for the company that made the software)? 
+
+:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: instructor
+
+Inline instructor notes: if Zenodo is unavailable ...
```
[the remark this answers is not in the document]
Comment (drcandacemakedamoore) [reply episodes/introduction_to_generative_ai.md]: Fixed.

**Pull request merged**